<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExistingDocumentDialog</class>
 <widget class="QDialog" name="AddExistingDocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add existing document</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../U2Gui.qrc">:/core/images/add_existing_document.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>595</width>
       <height>131</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="documentURLLabel">
        <property name="text">
         <string>Document location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="documentURLEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="documentURLButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="forceFormatLabel">
        <property name="text">
         <string>Document format:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="forceFormatCombo"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="forceReadOnlyCheck">
        <property name="text">
         <string>Force read-only mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>201</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QPushButton" name="formatSettingsButton">
        <property name="text">
         <string>Custom settings</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="save2DiskCheck">
        <property name="text">
         <string>Save file to disk before opening</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>421</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../U2Gui.qrc"/>
  <include location="../../../U2Gui.qrc"/>
  <include location="../../../core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>AddExistingDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
